models/classes/runner/synchronisation/action/Move.php

```diff
@@ -25,6 +25,8 @@
 use common_exception_InconsistentData;
```

Comment:
Repeat previous comment - Are these Exceptions used in this file not only in phpDoc goals?